z2jh.yml

```diff
@@ -469,7 +469,7 @@
         changed_when: False
 
       - name: Install JupyterHub release
-        shell: helm install {{ namespace }}-jupyterhub {{ helm_chart_repo_name }}/jupyterhub \
+        shell: helm upgrade --install {{ namespace }}-jupyterhub {{ helm_chart_repo_name }}/jupyterhub \
```

Comment:
is there a reason for this change. upgrade shouldn't be required the first time.